src/VisualStudio/Core/Def/Implementation/Library/AbstractLibraryService.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.VisualStudio.LanguageServices.Implementation.Library.VsNavInfo;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.Library
+{
+    internal abstract class AbstractLibraryService : ILibraryService
+    {
+        public Guid LibraryId { get; }
+        public __SymbolToolLanguage PreferredLanguage { get; }
+
+        public SymbolDisplayFormat TypeDisplayFormat { get; }
```

Comment:
Yes. VB shows return types in Object Browser and Class View while C# doesn't. There are a couple of other minor differences too IIRC.